daffodil-core/src/main/scala/org/apache/daffodil/core/grammar/primitives/PrimitivesTextNumber.scala

```diff
@@ -586,6 +597,29 @@ case class ConvertTextStandardNumberPrim(e: ElementBase)
         Nope
       }
 
+    // ICU does not have a way to set the pad position to after an affix if the positive pattern
+    // does not have that affix. For example, "* 0", will always have a pad position of
```

Comment:
Needs release note. This is a backward incompatible change (very minor one) because a "*0" in the negative pattern would have been always ignored before, but now means something that changes the behavior.

The warnings suggested in [DAFFODIL-2870](https://issues.apache.org/jira/browse/DAFFODIL-2870) are still meaningful, they just need to allow for the cases where the pad specifier in the negative pattern is used rather than ignored.

Reply:
Where do we capture and record a phrase or sentence which needs to be added to the release notes so we don't forget to include it later?